src/nupic/os/Env.cpp

```diff
@@ -126,6 +126,8 @@ char ** Env::environ_ = nullptr;
 
 #if defined(NTA_OS_DARWIN)
 #include <crt_externs.h>
+#elif defined(NTA_OS_WINDOWS)
+_CRTIMP extern char **_environ;
```

Comment:
It's fine for 0.2.0 and a usual way of handling this in a Windows app. Code requires it, a rewrite using replacement functions can be done later. Don't see a problem leaving like this, for now.